tasks/collectors/run-collectors/run-collectors.yaml

```diff
@@ -4,7 +4,7 @@ kind: Task
 metadata:
   name: run-collectors
   labels:
-    app.kubernetes.io/version: "0.3.0"
+    app.kubernetes.io/version: "0.4.0"
```

Comment:
Nit: It's a breaking change, so should be 5.0.0

Reply:
changed to be `1.0.0`